bgpd/bgp_ecommunity.h

```diff
@@ -202,4 +202,13 @@ extern void bgp_remove_ecomm_from_aggregate_hash(
 					struct ecommunity *ecommunity);
 extern void bgp_aggr_ecommunity_remove(void *arg);
 
+
+static inline void ecommunity_strip_rts(struct ecommunity *ecom)
+{
+	uint8_t subtype = ECOMMUNITY_ROUTE_TARGET;
+
+        ecommunity_strip(ecom, ECOMMUNITY_ENCODE_AS, subtype);
+        ecommunity_strip(ecom, ECOMMUNITY_ENCODE_IP, subtype);
+        ecommunity_strip(ecom, ECOMMUNITY_ENCODE_AS4, subtype);
```

Comment:
tabs please

Reply:
something happened to my settings, fixed now and pushed